browser/js/topics/topic.js

```diff
@@ -23,7 +23,7 @@ app.controller('TopicCtrl', function ($scope, TopicFactory, topic, VoteFactory,
 
```

Comment:
This file is also pretty large, I'd recommend splitting it up. Large files can 1) make code hard to find, and 2) lead to unnecessary merge conflicts when two people are working on two independent features but happen to touch the same file.